FirstDeliverable/CSV/csv.go

```diff
@@ -0,0 +1,67 @@
+package csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+type Item struct {
+	ItemID   string `json:"item_id"`
+	ItemName string `json:"item_name"`
+	ItemType string `json:"item_type"`
+}
+
+func ReadCSV() ([][]string, error) {
+	// Open the file
+	csvfile, err := os.Open("../FirstDeliverable/CSV/result.csv")
+	if err != nil {
+		log.Fatalln("Couldn't open the csv file", err)
+	}
+	defer csvfile.Close()
+
+	// Parse the file
+	reader := csv.NewReader(csvfile)
+	reader.LazyQuotes = true
+	reader.FieldsPerRecord = -1
+	records, err := reader.ReadAll()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
As a best practice, exit the application must happen only in the **main.go** file, so here, you should return the error and handle it in the main file